tfhe/src/integer/gpu/server_key/radix/ilog2.rs

```diff
@@ -330,6 +331,21 @@ impl CudaServerKey {
         res
```

Comment:
just for consistency, i think unchecked_trailing_zeros should be calling unchecked_trailing_zeros_async